draft-packages/empty-state/KaizenDraft/EmptyState/EmptyState.tsx

```diff
@@ -95,11 +97,7 @@ export const EmptyState: React.VFC<EmptyStateProps> = ({
       </div>
       <div className={styles.textSide}>
         <div className={styles.textSideInner}>
-          <Heading
-            variant="heading-3"
-            classNameOverride={styles.heading}
-            tag="div"
-          >
+          <Heading variant={headingVariant} classNameOverride={styles.heading}>
```

Comment:
Love the A11Y considerations in this PR and introducing more flexibility intro the component. One important call out in `<Heading />` is that `tag` controls the semantic tag type, whereas the `variant` is the styling. If `variant` is the only prop that is provided the component will attempt to infer the tag type.

The issue that could arise from this is that if we are not on a page with a h1 or h2 tag - we would be forced into a situation where we could not use h3 styling without breaking the heading level hierarchy. We could instead have another prop for the tag type that uses `AllowedHeadingTags` so that the user can specify the tag without impacting the style.

Reply:
Also a consideration about the h3 being default tag type - In this instance I'd err on the side of keeping the default tag to a `div` as it was originally. While it may be correct on most pages to default to h3, on some pages and h2 or even h1 may be more appropriate dependent on the content of the page and where its used in the product. 

If released as a non-breaking change that could also cause a number of accessibility bugs to arise.

Reply:
Ah great point!! I have just pushed a change so that the fall back is a div with the "heading-3" styling 👍